docs/source/intro.rst

```diff
@@ -1,24 +1,62 @@
 This is done by transforming storage on the Open Science Framework (OSF) into a `git-annex`_ repository.
 
 What can I use this extension for?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+----------------------------------
+
+You can use this extension to publish, store, collaborate on, or share your dataset (data) via the OSF_.
+Here is some inspiration on what you could do:
+
+- Publish your study (including its version history, data, code, results, and provenance) as a DataLad dataset to the OSF.
+  Share the project's OSF URL with colleages and collaborators to give them easy access to your work with a single ``datalad clone``.
+- Clone a friend's dataset -- from the OSF!
+- Use the OSF as a `special remote`_ to store data in the annex of your dataset.
+  With this, you can publish a dataset to `GitHub`_ or similar Git repository hosting services, and have your data published to the OSF (via an automatically set up publication dependency).
+  Your dataset will be exposed and available on GitHub, while data is stored in the OSF and data retrieval happens underneath the hood.
+- Take a version snapshot of all of your dataset's files and export them to the OSF.
+  This published one version of your project in a human-readable fashion to the OSF to make it available to the outside world.
 
-You can use this extension to use the OSF as a special remote to store data in the annex of a dataset.
-With this, you can `datalad publish` a dataset to GitHub or similar services and the data to the OSF (via a publication dependency).
-The extension is most beneficial for easy access to data stored on OSF via GitHub.
-If you are sharing your dataset via OSF and code via GitHub, this will allow smooth integration of both along with unified version management provided by DataLad.
+``datalad-osf`` comes with a range of hidden convenience functions for OSF interactions.
+Specifically, you will not need to create OSF projects via the OSF webinterface -- given appropriate credentials, ``datalad create-sibling-osf`` will create new projects under your user account and report back the generated URL.
+The :ref:`Tutorial` contains demonstrations of all functionality.
 
 What can I **not** use this extension for?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+------------------------------------------
 
-This tool does not work for data that is stored in a storage service other than OSF.
-Please refer to the list of `special remotes`_ as hosted by the git-annex website for other storage services.
+This tool does not work for data that is stored in a storage service other than the OSF_, and within the OSF, only OSF storage, no other third party storage, is supported.
+Please refer to the list of `special remotes`_ as hosted by the `git-annex`_ website for other storage services and how to use them with DataLad.
+Also, be mindful that OSF storage imposes a maximum file size of 5GB.
+Individual files larger than 5GB can not be published with this extension.
```

Comment:
FTR: We could (should?) actually tweak the special remote config to reflect that max filesize and limit chunks in the annex to 5GB or less.

Reply:
https://github.com/datalad/datalad-osf/issues/116

Reply:
indeed!